R/dataProcessPlots.R

```diff
@@ -375,14 +473,17 @@ dataProcessPlots = function(
   groupName = data.frame(RUN = c(0, lineNameAxis) + groupAxis / 2 + 0.5,
                          ABUNDANCE = rep(y.limup - 1, length(groupAxis)),
                          Name = levels(tempGroupName$GROUP))
-  
-  savePlot(address, "QCPlot", width, height)
+  if (isPlotly == FALSE) {
+      savePlot(address, "QCPlot", width, height)
+  }
+  plots <- list()
```

Comment:
Let's not use this way list();c(list, <element>). Instead, create a list of fixed length and then populate it

Reply:
Commit [better list handling](https://github.com/Vitek-Lab/MSstats/pull/107/commits/5c098d0c5d50d0300d8bca65672c0d6e9ec382e7)